rskj-core/src/main/java/co/rsk/db/importer/provider/BootstrapDataVerifier.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2019 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.db.importer.provider;
+
+import co.rsk.db.importer.BootstrapImportException;
+import co.rsk.db.importer.provider.index.data.BootstrapDataEntry;
+import co.rsk.db.importer.provider.index.data.BootstrapDataSignature;
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.crypto.ECKey;
+
+import java.math.BigInteger;
+import java.util.Map;
+
+public class BootstrapDataVerifier {
+
+    public int verifyEntries(Map<String, BootstrapDataEntry> selectedEntries) {
+        int verifications = 0;
+        if (selectedEntries.isEmpty()) {
+            return 0;
+        }
+        String verifiedHash = selectedEntries.values().iterator().next().getHash();
```

Comment:
`verifiedHash` is quite the name. The hash still has not been verified by anyone, it is just `THE hash`. Like my other comment, a think a cleaner solution is just have `hash` and check the signatures against those. The if condition is nice, but should not be the main defense :)

Reply:
Thx for the comment I improved this